src/components/image/image.ts

```diff
@@ -501,7 +507,7 @@ export class SbbImageElement extends LitElement {
               ];
             })}
             <img
-              alt=${this.alt || nothing}
+              alt=${this.alt || ''}
```

Comment:
Can you add a short comment before this line on why it is necessary in this case? thank you :)

Reply:
Sure, I just added the explanation.

Reply:
Thanks, nicely written.